src/types/index.ts

```diff
@@ -84,21 +84,44 @@ type UserTransformationLibrary = {
   VersionID: string;
 };
 
-type ProcessorTransformationRequest = {
-  request?: object;
-  message: object;
-  metadata: Metadata;
-  destination: Destination;
-  libraries: UserTransformationLibrary[];
-};
+type RudderTimestamp = Date | string;
+type RudderMessageType =
+  | 'identify'
+  | 'track'
+  | 'page'
+  | 'screen'
+  | 'group'
+  | 'alias'
+  | 'audiencelist';
+
+type RudderMessageSchema = Partial<{
```

Comment:
there's `RudderMessage` too - https://github.com/rudderlabs/rudder-transformer/blob/e4c6bb27591672ba369750c6a1c03bfa600fe4d7/src/types/index.ts#L178
Do we need both of them ?

Reply:
I used `RudderMessage` itself but there were too many changes that were needed in order for the tests to pass & compile. So, kind of yes, we need both of them.
[Backlog] We have to plan to get rid of both of these types & stick with only one type.